eng/common/templates/steps/generate-sbom.yml

```diff
@@ -5,7 +5,7 @@
 # IgnoreDirectories - Directories to ignore for SBOM generation. This will be passed through to the CG component detector.
 
 parameters:
-  PackageVersion: 7.0.0
+  PackageVersion: 8.0.0
```

Comment:
Since this is in eng/common, I don't think it needs to be merged between branches.  It should come in as part of the dotnet/arcade dependency update.